client/src/app/ui/modules/editor/editor.module.ts

```diff
@@ -16,13 +16,15 @@ import { EditorEmbedDialogComponent } from './components/editor-embed-dialog/edi
 import { EditorHtmlDialogComponent } from './components/editor-html-dialog/editor-html-dialog.component';
 import { EditorImageDialogComponent } from './components/editor-image-dialog/editor-image-dialog.component';
 import { EditorLinkDialogComponent } from './components/editor-link-dialog/editor-link-dialog.component';
+import { TabNavigationDirective } from './directives/tab-navigation.directive';
 
 const DECLARATIONS = [
     EditorComponent,
     EditorImageDialogComponent,
     EditorEmbedDialogComponent,
     EditorLinkDialogComponent,
-    EditorHtmlDialogComponent
+    EditorHtmlDialogComponent,
+    TabNavigationDirective
```

Comment:
Align naming or if not editor specific move to `ui/directives`. 

```suggestion
    EditorTabNavigationDirective
```